propel/resource_data_pool.go

```diff
@@ -103,6 +103,38 @@ func resourceDataPool() *schema.Resource {
 				ForceNew:    true,
 				Description: "The Data Pool's timestamp column.",
 			},
+			"cursor": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The column to track whether a record should be synced. An example of a cursor would be a timestamp column like `updated_at`",
+			},
+			"syncing": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Computed:    true,
+				Description: "The Data Pool's syncing settings.",
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"status": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Indicates whether syncing is enabled or disabled.",
+						},
```

Comment:
Would we want to allow setting this? I think eventually, yes, right?

Reply:
yes, we want to add support for pausing the Data Pool syncing but will do it in a diff PR